src/csvcubed/models/cube/qb/components/concept.py

```diff
@@ -38,22 +47,71 @@ def __eq__(self, other):
     def __hash__(self):
         return self.code.__hash__()
 
+    def validate_data(
+        self,
+        data: pd.Series,
+        column_csvw_name: str,
+        csv_column_uri_template: str,
+        column_csv_title: str,
+    ) -> List[ValidationError]:
+        return []
+
+    def _get_validations(self) -> Dict[str, ValidationFunction]:
+
+        return {
+            "label": validate_str_type,
+            "code": validate_str_type,
+            "parent_code": validate_optional(validate_str_type),
+            "sort_order": validate_optional(validate_str_type),
+            "description": validate_optional(validate_str_type),
+            **UriIdentifiable._get_validations(self),
+        }
+
 
 @dataclass
 class ExistingQbConcept(SecondaryQbStructuralDefinition):
     """Represents a QbConcept which is already defined at the given URI."""
 
     existing_concept_uri: str
 
-    _existing_concept_uri_validator = validate_uri("existing_concept_uri")
+    _existing_concept_uri_validator = pydantic_validate_uri("existing_concept_uri")
+
+    def validate_data(
```

Comment:
Again, get rid of this `validate_data` method.

Reply:
Removed it